lib/v2/gov/hunan/changsha/major-email.js

```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const iconv = require('iconv-lite');
+const baseUrl = 'http://wlwz.changsha.gov.cn';
+
+module.exports = async (ctx) => {
+    const listPage = await got.get('http://wlwz.changsha.gov.cn/webapp/cs2020/email/index.jsp', {
+        responseType: 'buffer',
+    });
+    listPage.data = iconv.decode(listPage.data, 'gbk');
+    const $ = cheerio.load(listPage.data);
+    const list = $('.table1 tbody tr')
+        .slice(1)
+        .map((_, tr) => {
+            tr = $(tr);
+
+            return {
+                title: tr.find('td[title]').attr('title'),
+                link: baseUrl + tr.find('td[title] > a').attr('href'),
+                author: tr.find('td:last').text(),
+            };
+        })
+        .get();
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const postPage = await got.get(item.link, {
+                    responseType: 'buffer',
+                });
+                postPage.data = iconv.decode(postPage.data, 'gbk');
+                const $ = cheerio.load(postPage.data);
+
+                const data = {
+                    title: item.title,
+                    description: $('.letter-details').html().trim(),
+                    pubDate: new Date($('.letter-details div:first table tr:nth-child(2) > .td_label2').text() + ' GMT+8'),
```

Comment:
You can use `parseDate()` and `timezone()` instead
docs: https://docs.rsshub.app/joinus/pub-date.html